src/containers/CreateStreamMainCard/index.tsx

```diff
@@ -201,6 +203,7 @@ const CreateStream = () => {
                     {...field}
                     className="w-[236px]"
                     label="End date"
+                    toolTipTitle="End date"
                     details="By specifying the end date of your stream, the total amount to be streamed will be calculated."
```

Comment:
move all the details to a separate file called `/src/constants/tooltipDetails`.

The structure of the file should be:

```javascript
export default {
    createStream: {
        endDate: 'By specifying the end date of your stream...',
        cliffDate: 'Cliff time specifies...'
   }
}
```

import these properties to the create stream and use it like

```tooltipDetails={tooltilDetails.createStream.startDate}```